### DISCUSSION TITLE
Automatically hide +1 and 👍 responses

### DISCUSSION THREAD

**Discussion by suprstarrd** [community/community#80962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

[This comment section is FULL of only +1 responses.](https://github.com/orgs/community/discussions/7923). I'd like to be able to hide those (and still see those that have any other form of text in it, like an image or any other remark!)

**Comment by suprstarrd** [comment]
Ironically(?), I'll take any +1's I can get here 😆

**Reply by baartch** [reply]
+1

**Comment by alkanna** [comment]
+1

**Comment by GameLord2011** [comment]
👍

**Comment by stephendotgg** [comment]
+1

**Comment by hifzi** [comment]
+1

**Comment by baartch** [comment]
+1
